src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -515,15 +515,15 @@ f1_accumdepr_prvsn,4,1994,(413) Exp. of Elec. Plt. Leas. to Others,reported_valu
 f1_accumdepr_prvsn,5,1994,Transportation Expenses-Clearing,reported_value,transportation_expenses_clearing,,
 f1_accumdepr_prvsn,6,1994,Other Clearing Accounts,reported_value,other_clearing_accounts,,
 f1_accumdepr_prvsn,7,1994,Other Accounts (Specify):,reported_value,other_accounts,,
-f1_accumdepr_prvsn,8,1994,Line 8,reported_value,other_accounts_details,TRUE,DBF contains unstructured data with a description in the item column.
+f1_accumdepr_prvsn,8,1994,Line 8,reported_value,other_accounts_details,True,DBF contains unstructured data with a description in the item column.
```

Comment:
there are several scattered changes throughout this CSV: `TRUE` -> `True` and `#N/A` -> `""`

this happened bc I used the XBRL taxonomy viewer thing to do the mapping bc it is way easier/less error prone imo. But it is all in TitleCase, so I read in the csv, replaced with snake_case and saved.